modules/node/src/NodeAccessControlHandler.php

```diff
@@ -27,17 +28,49 @@ class NodeAccessControlHandler extends EntityAccessControlHandler implements Nod
    */
   protected $grantStorage;
 
+  /**
+   * Node entity storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * Map of revision operations.
+   *
+   * Keys contain revision operations, where values are an array containing the
+   * permission operation and supplemental entity operation.
+   *
+   * Permission operation is used to build the required permission, e.g.
+   * 'permissionOperation all revisions', 'permissionOperation type revisions'.
+   *
+   * Supplemental entity operation is used to determine access, for
+   * 'delete revision' operation, an account must also have access to 'delete'
+   * operation on an entity.
+   *
+   * @internal Only to be used internally by this class. Consider as protected.
+   */
+  const REVISION_OPERATION_MAP = [
```

Comment:
is this supported in all versions of php we target? assuming 8.8 is 7.0 then answer is yes

Reply:
@larowlan you mean const as array? 5.6: https://3v4l.org/Dt0Af